docs/cudf/source/user_guide/data-types.md

```diff
@@ -16,11 +16,11 @@ All data types in cuDF are [nullable](missing-data).
 | Floating-point       | `'float32'`, `'float64'`                                                                          |
 | Datetime             | `'datetime64[s]'`, `'datetime64[ms]'`, `'datetime64['us']`, `'datetime64[ns]'`                    |
 | Timedelta (duration) | `'timedelta[s]'`, `'timedelta[ms]'`, `'timedelta['us']`, `'timedelta[ns]'`                        |
-| Category             | {py:func}`cudf.CategoricalDtype`                                                                  |
+| Category             | `cudf.CategoricalDtype`                                                                           |
```

Comment:
Looks like there are now dedicated pages for `cudf.CategoricalDtype`, `cudf.ListDtype`, `cudf.StructDtype`, `cudf.Decimal...Dtype`, is it possible to link them here? Right now they seem to be appearing as text only here.

Reply:
Hmm I wonder why I'm not seeing warnings. Let me see what's going on here.

Reply:
I needed to add in the roles again so that they would be found. Everything seems to be working again now. For some reason it requires fully qualified names for the dtypes (`cudf.CategoricalDtype` doesn't work because it's in the inventory as `cudf.core.dtypes.CategoricalDtype`), which is unfortunate but something that I think we should address in a follow-up.